src/java.base/share/classes/java/math/BigDecimal.java

```diff
@@ -4308,6 +4314,7 @@ private static BigInteger expandBigIntegerTenPowers(int n) {
         1000000000000000000L   // 18 / 10^18
     };
 
+    @Stable
```

Comment:
This one is a bit dangerous as it can be reassigned. I think we should not mark this array @Stable unless we have analysed in detail the implications thereof.